docs/docs/Sprint 4/back_end.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Back-End - API
+sidebar_position: 2
+---
+
+O objetivo principal dessa página é documentar a criação e implementação de uma API realizada pelo grupo, visando interagir com o Banco de Dados existente. Para uma implementação eficaz da API, foram utilizadas tecnologias que estão em alta no mercado e que coiencidiam com os objetivos do grupo, de manter a aplicação com uma manutenibilidade fácil para os desenvolvedores, porém ainda tendo eficiência em realizar consultas ao banco. Com isso, foi desenvolvida uma aplicação **express**, juntamente aos **websockets** já existentes.
```

Comment:
acho válido add o porque de ser uma api e não junto do backend, pq teve q ser separado (é p poder fornecer os dados p outras áreas da empresa, certo, p funcionar como consulta? se n for isso e eu tiver maluca só ignora o comentário)

Reply:
Concordo